README.md

```diff
@@ -109,6 +109,14 @@ commit your own `.semgrepignore`.
 Note that `.semgrepignore` is picked up only by the action,
 and will not be honored when running `semgrep` manually.
 
+### Audit Mode
```

Comment:
Looks much better! Why might someone want to use this feature and/or what outcome does it allow them to achieve? I'd love to articulate that answer here for readers.